packages/migrate/src/MigrateEngine.ts

```diff
@@ -139,15 +140,11 @@ export class MigrateEngine {
           this.listeners[result.id](result)
           delete this.listeners[result.id]
         }
-      } else {
```

Comment:
Deleted because I'm confident this code path is never taken, based on what the ME returns.